declarations/Tchap.json

```
@@ -11,8 +11,12 @@
     "Terms of Service": {
       "fetch": "https://www.tchap.gouv.fr/cgu/",
```

The [following error](https://github.com/OpenTermsArchive/contrib-declarations/actions/runs/4743549444/jobs/8423240061?pr=816) could be fix by updating the fetch URL.
```suggestion
      "fetch": "https://tchap.beta.gouv.fr/cgu",
```